lib/Round.js

```diff
@@ -0,0 +1,34 @@
+class Round {
```

Comment:
Really great work here. Based on our conversation this morning, I can go either way with including the Game class. We could break out that Game handles the creation of the categories and the clues and saves in array that the Round class then calls on. Lets align today!